test/e2e/framework/util.go

```diff
@@ -836,6 +836,18 @@ func WaitForDefaultServiceAccountInNamespace(c *client.Client, namespace string)
 	return waitForServiceAccountInNamespace(c, namespace, "default", ServiceAccountProvisionTimeout)
 }
 
+// WaitForFederationApiserverReady waits for the federation apiserver to be ready.
+// It tests the readiness by sending a GET request and expecting a non error response.
+func WaitForFederationApiserverReady(c *federation_internalclientset.Clientset) error {
+	return wait.PollImmediate(time.Second, 1*time.Minute, func() (bool, error) {
```

Comment:
Please use one of the standard timeouts in utils. 

https://github.com/kubernetes/kubernetes/blob/master/docs/devel/writing-good-e2e-tests.md

Perhaps ServiceRespondingTimeout?

https://github.com/kubernetes/kubernetes/blob/master/test/e2e/framework/util.go#L131